helm_deploy/court-case-service/templates/_envs.tpl

```diff
@@ -8,7 +8,7 @@ env:
     value: "{{ .Values.image.port }}"
 
   - name: SPRING_PROFILES_ACTIVE
-    value: "postgres,instrumented"
```

Comment:
Note: the postgres profile was removed ages back because it didn't actually do anything so I've removed it from the  values files accordingly